StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/PaymentSheet+API.swift

```diff
@@ -35,6 +35,17 @@ extension PaymentSheet {
         paymentHandler: STPPaymentHandler,
         completion: @escaping (PaymentSheetResult) -> Void
     ) {
+
+        // If processing a deferred intent, store state for later processing
```

Comment:
Open to suggestions here on the singleton. Since the load/confirm is static we need to save it in a static variable for later use.

Reply:
Hrm I can see how this is convenient, but we shouldn't need it - we can flow everything we need across the various methods called in the deferred flow.